en-US/foundation/advocacy.lang

```diff
@@ -0,0 +1,90 @@
+## NOTE: demo page at https://www-dev.allizom.org/foundation/advocacy/
+## URL: https://www-dev.allizom.org/%LOCALE%/foundation/advocacy/
+
+
+;Mozilla Advocacy
+Mozilla Advocacy
+
+
+;Rallying citizens to protect the open Internet
+Rallying citizens to protect the open Internet
+
+
+;Mozilla’s advocacy engine works with allies to cultivate a global force of tens of millions of people, who are aware of the issues that are vital to the future of the open Internet and prepared to take action in support of the rights of citizens of the web.
+Mozilla’s advocacy engine works with allies to cultivate a global force of tens of millions of people, who are aware of the issues that are vital to the future of the open Internet and prepared to take action in support of the rights of citizens of the web.
+
+
+;We are dedicated to building a world where:
+We are dedicated to building a world where:
+
+
+;People are making more conscious choices to build and protect the open Internet
+People are making more conscious choices to build and protect the open Internet
+
+
+;Companies and governments create products, policies and laws to support the open Internet because of public pressure to do so
+Companies and governments create products, policies and laws to support the open Internet because of public pressure to do so
+
+
+;How we work: A focus on depth and scale
+How we work: A focus on depth and scale
+
+
+;Mozilla’s advocacy approach incorporates both scale to depth and depth to scale to properly address the diversity of issues that we work on.
+Mozilla’s advocacy approach incorporates both scale to depth and depth to scale to properly address the diversity of issues that we work on.
+
+
+;This approach enables Mozilla to simultaneously build our capacity to mobilize millions of people quickly and grow our organizing power in local communities, which will make it possible to win on our issues in the long-term.
+This approach enables Mozilla to simultaneously build our capacity to mobilize millions of people quickly and grow our organizing power in local communities, which will make it possible to win on our issues in the long-term.
+
+
+;We also push ourselves to be proactive—setting the topic and tone of the conversation, and promoting solutions—as well as reactive to current events or undesirable policies when necessary.
+We also push ourselves to be proactive—setting the topic and tone of the conversation, and promoting solutions—as well as reactive to current events or undesirable policies when necessary.
```

Comment:
These - without spaces before and after don't look right (they have them in the leadership file).